roles/confluent.kafka_broker/defaults/main.yml

```diff
@@ -73,7 +73,7 @@ kafka_broker:
     zookeeper.connection.timeout.ms: 18000
     confluent.license.topic.replication.factor: "{{kafka_broker_default_interal_replication_factor}}"
     confluent.metadata.topic.replication.factor: "{{kafka_broker_default_interal_replication_factor}}"
-    metric.reporters: io.confluent.metrics.reporter.ConfluentMetricsReporter
+    metric.reporters: "{% if kafka_broker_metrics.enabled|bool %}io.confluent.metrics.reporter.ConfluentMetricsReporter{% endif %}"
```

Comment:
how does metric.reporter show up in server.properties if kafka_broker_metrics.enabled=false?

Reply:
As such:
`metric.reporters=`